src/qt/forms/debugwindow.ui

```diff
@@ -592,34 +592,17 @@
           <number>3</number>
          </property>
          <item>
-          <widget class="QPushButton" name="promptIcon">
-           <property name="enabled">
-            <bool>false</bool>
-           </property>
+          <widget class="QLabel" name="promptIcon">
```

Comment:
`GUIUtil::ThemedLabel`?

Reply:
See, a rough implementation of this suggestion here: https://github.com/jarolrod/gui/commit/4bc138e647cd6d5c7b2671d070c8c15623d1c37f

I think the current approach is simpler

Reply:
not applicable anymore since 2f23ad2